perllib/FixMyStreet/DB/Result/Comment.pm

```diff
@@ -290,6 +300,10 @@ sub meta_line {
 
     }
 
+    if ($self->get_extra_metadata('defect_raised')) {
+        $update_state .= ( $update_state ? ', ' : '' ) . _( 'and a defect raised' );
```

Comment:
Is there a situation where update_state can't be set here? It doesn't feel like there should be.